modules/installation-configuration-parameters.adoc

```diff
@@ -1007,6 +1007,17 @@ For a private cluster, specify a private subnet for each availability zone.
 For clusters that use AWS Local Zones, you must add AWS Local Zone subnets to this list to ensure edge machine pool creation.
 |Valid subnet IDs.
 
+|platform:
+  aws:
+    publicIpv4Pool:
+|The public IPv4 pool ID that is used to allocate Elastic IPs (EIPs) when `publish` is set to `External`. You must provision and advertise the pool in the same AWS account and region of the cluster. You must ensure that you have 2n + 1 IPv4 available in the pool where n is the total number of AWS zones used to deploy the Network Load Balancer (NLB) for API, NAT gateways, and bootstrap node. For more information about bring your own IP addresses (BYOIP) in AWS, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-byoip.html#byoip-onboard[Onboard your BYOIP].
+| A valid link:https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-public-ipv4-pools.html[public IPv4 pool id]
+
+[NOTE]
```

Comment:
Ideally parameterize AWS as attributes exist: 
![Screenshot from 2024-06-19 16-23-30](https://github.com/openshift/openshift-docs/assets/57954076/f215716b-9e67-4ebf-b49f-d4937e61b8ed)

Reply:
I see. Looks like the entire assembly doesn't use attributes for AWS. Thanks!